src/odemis/driver/test/tescan_test.py

```diff
@@ -240,7 +241,104 @@ def test_external(self):
         time.sleep(6)  # Wait for value refresh
         self.assertEqual(orig_ext, ebeam.external.value)
 
-    # TODO: test move
+    def test_move_rel(self):
+        """
+        Check it's possible to move the stage with relative moves
+        """
+        pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"x":2e-6, "y":3e-6})
+        f.result()
+        self.assertNotEqual(self.stage.position.value, pos)
+        time.sleep(6)  # wait until .position is updated
+        self.assertNotEqual(self.stage.position.value, pos)
+
+        f = self.stage.moveRel({"x":-2e-6, "y":-3e-6})
+        f.result()
+        testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+        time.sleep(6)
+        testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+
+        # Try a relative move outside of the range (less than min)
+        axes = self.stage.axes
+        # toofar = {"x": axes["x"].range[0] - pos["x"] - 10e-6}
+        # f = self.stage.moveRel(toofar)
+        # with self.assertRaises(ValueError):
+        #     f.result()
+        # testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+        # time.sleep(6)
+        # testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+        #
+        # # Try a relative move outside of the range (more than max)
+        # toofar = {"y": axes["y"].range[1] - pos["y"] + 10e-6}
+        # f = self.stage.moveRel(toofar)
+        # with self.assertRaises(ValueError):
+        #     f.result()
+        # testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+        # time.sleep(6)
+        # testing.assert_pos_almost_equal(self.stage.position.value, pos, atol=0.1e-6)
+
+        # Move really too far (more than the range)
+        with self.assertRaises(ValueError):
+            f = self.stage.moveRel({"x":axes["x"].range[1] - axes["x"].range[0] + 10e-6})
+
+    def test_move_abs(self):
+        """
+        Check it's possible to move the stage in absolute coordinates
+        """
+        p = self.stage.position.value.copy()
+        subpos = self.stage.position.value.copy()
+        subpos["x"] += 50e-6
+        f = self.stage.moveAbs(subpos)
+        f.result()
+        testing.assert_pos_almost_equal(self.stage.position.value, subpos)
+        time.sleep(6)
+        testing.assert_pos_almost_equal(self.stage.position.value, subpos)
+
+        self.stage.moveAbsSync({"x": p["x"]})
+        testing.assert_pos_almost_equal(self.stage.position.value, p)
+        time.sleep(6)
+        testing.assert_pos_almost_equal(self.stage.position.value, p)
+
+    def test_stop(self):
+        """
+        Check it's possible to move the stage
```

Comment:
Maybe update the docstring a little bit with some info about the testcode for stopping while moving the stage